app/pathogen/arbovirus/wip/page.tsx

```diff
@@ -30,8 +31,13 @@ export default function RedesignedArbovirusPage() {
   
   return (
     <div className="col-span-12 row-span-2 grid gap-0 grid-cols-12 grid-rows-2 grid-flow-col w-screen overflow-hidden border-box -my-4 -ml-4">
-      <div className="col-span-2 h-full row-span-2">
-        <div className="w-full h-full bg-orange-500"> placeholder for filters </div>
+      <div className="overflow-y-scroll col-span-2 h-full row-span-2 border-black border-r-2">
+        <Filters
+          className="p-4"
+          excludedFields={
+            idOfCurrentlyViewedSection === RedesignedArbovirusPageSectionId.MAP ? [FilterableField.pathogen] : [] 
+          }
```

Comment:
The map has a pathogen filter of it's own so we use `excludedFields` in the original to make sure there isn't two pathogen filtration places showing up at the same time. I'm just copying that behaviour here. When you stop looking at the map, the pathogen filter re-appears in the filter sidebar.
![image](https://github.com/PathoTracker/Pathotracker/assets/86806388/78122e86-5e6d-418b-9283-63293ecc8486)
![Peek 2024-01-21 17-14](https://github.com/PathoTracker/Pathotracker/assets/86806388/aec5b886-a200-4e3d-a178-c115a2045038)